src/os.cc

```diff
@@ -35,6 +35,7 @@ namespace toit {
 Mutex* OS::_global_mutex = null;
 Mutex* OS::_scheduler_mutex = null;
 Mutex* OS::_resource_mutex = null;
+int    OS::_cpu_revision = 999;
```

Comment:
Why is this a reasonable default value? -1? 0? Add a comment if 999 makes sense.